lib/i18n/tests/basics.rb

```diff
@@ -5,12 +5,11 @@ def teardown
         I18n.available_locales = nil
       end
 
-      test "available_locales returns the locales stored to the backend by default" do
+      test "available_locales returns the available_locales produced by the backend, by default" do
         I18n.backend.store_translations('de', :foo => 'bar')
         I18n.backend.store_translations('en', :foo => 'foo')
 
-        assert I18n.available_locales.include?(:de)
-        assert I18n.available_locales.include?(:en)
+        assert_equal I18n.backend.available_locales, I18n.available_locales
```

Comment:
Nit: usually we order assertions as `assert_equal(expected, actual)`
```suggestion
        assert_equal I18n.available_locales, I18n.backend.available_locales
```